pkgs/top-level/all-packages.nix

```diff
@@ -7138,6 +7138,14 @@ in
 
   nvchecker = with python3Packages; toPythonApplication nvchecker;
 
+  nvfetcher = with haskell.lib; overrideCabal (justStaticExecutables haskellPackages.nvfetcher) (drv: {
+    executableToolDepends = [ makeWrapper ];
+    postInstall = ''
```

Comment:
We could move this override to `configuration-nix.nix`, I think? then `haskellPackages.nvfetcher` would also be functional were someone to use it.

Reply:
I have remarked this in the corresponding PR. berberman said they will improve on this in a future PR they have planned anyways.